src/main/java/org/xwiki/contrib/macro/docaccordion/internal/DocAccordionMacro.java

```diff
@@ -369,7 +369,6 @@ private DocumentReference getSpaceAWMDataXClass(SpaceReference spaceReference) t
 
                 // Accordion item Panel heading title link
                 Map<String, String> accordionItemPanelHeadingTitleLinkParams = new HashMap<>();
-//                if (cssClasses == "") {
                 accordionItemPanelHeadingTitleLinkParams.put("class", "collapsed");
```

Comment:
The 'collapsed' class should be added when the item is closed and the first item is opened by default, so you will need to make a test to check if the current item is not the first item and if it is the first item, check if it is closed.

The test is 

`if(cssClasses == "") {
  accordionItemPanelHeadingTitleLinkParams.put("class", "collapsed");
}`

Thanks for your contribution